tfjs-backend-webgpu/src/kernels/Conv2D_impl.ts

```diff
@@ -270,7 +270,7 @@ export function conv2DImpl({
   const useVec4 =
       (convInfo.inChannels % 4 === 0 ||
        (convInfo.inChannels === 3 && convInfo.padInfo.type === 'VALID')) &&
-      convInfo.outChannels % 4 === 0 && convInfo.outChannels >= 32;
```

Comment:
Per our test, Conv2DMMVec4Program is always faster than Conv2DMMProgram based on current configuration. So it's not necessary to add this limitation. And without this limitation, the current test `fused conv2d webgpu {"WEBGPU_CPU_FORWARD":false} leakyrelu bias stride 2 x=[1,8,8,16] f=[3,3,16,1] s=[2,2] d=8 p=same` can catch this leakyRelu bug.